torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -1392,6 +1413,24 @@ struct PythonPrintImpl {
       for (auto& method : classType->methods()) {
         printFunction(*method);
       }
+#ifndef FBCODE_CAFFE2
+      // TODO Change to print hooks outside of module if same hook
+      // is being used on multiple modules and serialzed file size
+      // is bloating because of it
```

Comment:
I think you made an issue for this already so I'd prefer not to have a TODO in the code.